.github/workflows/tools.yml

```diff
@@ -20,13 +20,7 @@ jobs:
           - id: eslint
             subsystem: tools
             label: tools
-            run: |
-              NEW_VERSION=$(npm view eslint dist-tags.latest)
-              CURRENT_VERSION=$(node -p "require('./tools/node_modules/eslint/package.json').version")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
-                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
```

Comment:
We need to keep defining `NEW_VERSION` in the env somehow.

Reply:
Nice catch!! I deleted it because I thought I was introducing it my self ^^ I miss read it some how. Let me fix this.

Reply:
That being said @RaisinTen did not use it in this script 🤔

Reply:
@tony-go `NEW_VERSION` is already being used at the bottom of this file by the bot for creating the automated update PR - https://github.com/nodejs/node/blob/a5ede9ccf579e82881844462f8f506e171f54989/.github/workflows/tools.yml#L171-L175. I didn't use it in the `tools/dep_updaters/update-postject.sh` script because of the same reason why `tools/dep_updaters/update-eslint.sh` doesn't use it, i.e., it isn't strictly needed.